From 4bb023e40ac312db27012c97b8e51f124c9cd035 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Thu, 12 Feb 2015 12:47:47 +0200
Subject: [PATCH] makefile
Organization: Private
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 Makefile.in | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/Makefile.in b/Makefile.in
index e2fea02..90b80f2 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -28,7 +28,7 @@ exec_prefix = @exec_prefix@
 
 bindir = $(DESTDIR)@bindir@
 mandir = $(DESTDIR)@mandir@
-libdir = $(DESTDIR)@libdir@
+libdir = $(DESTDIR)/usr/share/bvi
 
 man1dir = $(mandir)/man1
 
@@ -65,7 +65,7 @@ bmore: $(BMOBJ)
 	$(CC) $(CFLAGS) $(DEFS) -c $<
 
 bmore.o: bmore.c
-	$(CC) $(CFLAGS) $(DEFS) -DHELPFILE=\"$(libdir)/bmore.help\" -c $<
+	$(CC) $(CFLAGS) $(DEFS) -DHELPFILE=\"/usr/share/bvi/bmore.help\" -c $<
 
 
 install: all installdirs
@@ -76,8 +76,8 @@ install: all installdirs
 	@$(INSTALL_DATA) bmore.help $(libdir)
 	@if [ -f $(bindir)/bview ]; then rm $(bindir)/bview; fi
 	@if [ -f $(bindir)/bvedit ]; then rm $(bindir)/bvedit; fi
-	@ln -s bvi $(bindir)/bview
-	@ln -s bvi $(bindir)/bvedit
+	cd $(bindir) && ln -s bvi bview
+	cd $(bindir) && ln -s bvi bvedit
 
 installdirs:
 	$(SHELL) ${srcdir}/mkinstalldirs $(bindir) $(libdir) $(man1dir)
-- 
2.1.4

