From 0645992da7bee289137dcf9ff283d3f4de9438e5 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Sat, 14 Sep 2013 17:27:54 +0300
Subject: [PATCH] Makefile.in: install target fix
Organization: Private
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 Makefile.in |   10 +++++-----
 1 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/Makefile.in b/Makefile.in
index e2fea02..5bee110 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -65,7 +65,7 @@ bmore: $(BMOBJ)
 	$(CC) $(CFLAGS) $(DEFS) -c $<
 
 bmore.o: bmore.c
-	$(CC) $(CFLAGS) $(DEFS) -DHELPFILE=\"$(libdir)/bmore.help\" -c $<
+	$(CC) $(CFLAGS) $(DEFS) -DHELPFILE=\"/usr/share/bvi/bmore.help\" -c $<
 
 
 install: all installdirs
@@ -73,14 +73,14 @@ install: all installdirs
 	@$(INSTALL_DATA) bvi.1 bmore.1 $(man1dir)
 	@$(INSTALL_PROGRAM) bvi $(bindir)
 	@$(INSTALL_PROGRAM) bmore $(bindir)
-	@$(INSTALL_DATA) bmore.help $(libdir)
+	@$(INSTALL_DATA) -D bmore.help $(DESTDIR)/usr/share/bvi/bmore.help
 	@if [ -f $(bindir)/bview ]; then rm $(bindir)/bview; fi
 	@if [ -f $(bindir)/bvedit ]; then rm $(bindir)/bvedit; fi
-	@ln -s bvi $(bindir)/bview
-	@ln -s bvi $(bindir)/bvedit
+	cd $(bindir) && ln -s bvi bview
+	cd $(bindir) && ln -s bvi bvedit
 
 installdirs:
-	$(SHELL) ${srcdir}/mkinstalldirs $(bindir) $(libdir) $(man1dir)
+	$(SHELL) ${srcdir}/mkinstalldirs $(bindir) $(man1dir)
 
 uninstall:
 	@echo "Uninstalling bvi"
-- 
1.7.9

